src/services/auth.js

```diff
@@ -48,6 +49,20 @@ passport.use(new HTTPSignatureStrategy(
       })
   }))
 
+passport.use(new ClientCertStrategy((userName, info, done) => {
+  if (!config.auth.client_certificates_enabled) {
+    return done(new UnauthorizedError('Unsupported authentication method'))
+  }
+
+  Account.findByName(userName)
+    .then(function (userObj) {
+      if (!userObj || userObj.is_disabled || userObj.fingerprint !== info.fingerprint) {
```

Comment:
There is still some discussion over whether the `userName` will be the fingerprint. For now, we can assume we're either using the fingerprint itself as the username or, the Common Name as the username and verifying the fingerprint. The fingerprint will have to be communicated out of band. Once the first point is settled, I'll update [this PR](https://github.com/interledger/five-bells-shared/pull/50) and then follow that with updating the strategy here